### ISSUE TITLE
Which version of patchtst do you use in your exp —supervised or self-supervised?"

### ISSUE THREAD

**Issue by iganggang** [gaasher/embedding_thesis#2]

**Comment by iganggang** [comment]
i can not find it in your code. thank u very much. @gaasher

**Comment by iganggang** [comment]
i guess that maybe u use supervised version.

**Comment by iganggang** [comment]
In your patchTST.py code, 
57 batch_size, num_channels, seq_len, patch_len = x.shape
i think it should be num_patches, not seq_len.
Could u give me a reply?

**Comment by iganggang** [comment]
if mode == 'cnn':
          self.embed = nn.Sequential(
            nn.Conv1d(in_channels=1, out_channels=8, kernel_size=3, padding=1),
            nn.ReLU(),
            nn.Conv1d(in_channels=8, out_channels=32, kernel_size=3, padding=1),
            nn.ReLU(),
        )
 
In this code, how do you choose the parameters out_channels and out_channels?
In Nie's patchtst script etth1, d_model is 16, how to choose them? 
according to your paper, out_channels=1, but the results are not good.
       then, I read your paper, I still feel confused. Could you give a reply? thank u very much.

**Comment by iganggang** [comment]
would you give me a reply?
